### ISSUE TITLE
[FEATURE REQUEST] Disable the Advancement when joining a new world.

### ISSUE THREAD

**Issue by szunakabeats** [Alex-the-666/CloudStorage#71]
can you please add a option to disable the first cloud storage advancement when joining a new world ? it takes the immersivenes away because i have a starting message for the player when joining a world. and the message gets pushed up because of the advancement.

thx for your time and the hard work behind the mod!

**Comment by joe-cheem** [comment]
Wanted the same thing so I just edited the advancement in the source and rebuilt the .jar
Here you go buddy: https://www.mediafire.com/file/ezfv634srponh0k/cloudstorage-1.4.0-1.20.jar/file

**Comment by szunakabeats** [comment]
well thx for your work, but the problem is, everytime a update releases i have to do this again. so ill just w8 and see if alex implements this in the mod itself :)

**Comment by joe-cheem** [comment]
fair enough, though looking at the state of the mod it seems that it's not being worked on anymore, hence why I thought of doing the fix myself :)

**Comment by MarioSMB** [comment]
Just a heads up, you don't need to edit the .jar file to disable the advancement! You just need to make a datapack and load it with the world generation.  
That said, @AlexModGuy seems to have included such messages in several of his mods and it is rather intrusive, so I hope some activity returns to these mods and the messages can be disabled properly by default.   

We know we have the mod installed and we're glad to have it, you don't need to remind us like this!!
